### DISCUSSION TITLE
Multi-database app with PocketBase as a framework

### DISCUSSION THREAD

**Discussion by vipas84** [pocketbase/pocketbase#5138] [Q&A] [answered]
I'm considering creating a SAAS app. Within this app I need to have a separate sqliteDB per each client.
What could be the best approach to do so with Pocketbase. Am I going to face current PocketBase architectural limitations?

**Comment by whmeitzler** [comment, marked as answer]
Reworking pocketbase to support many db connections is technically feasible, and almost assuredly not worth it as you're going to be supporting a Frankenstein.

If you're looking for true isolation among clients, consider spinning up multiple instances of pocketbase (each with its own db), one per customer. The beauty of Pocketbase is its ease of deployment. Leverage that!


If you're looking for a single binary deployment, you could explore if your clients use cases fit into a unified data model.


If it were me, I'd deploy multiple instances. Maybe check out pockethost.io?

**Comment by mukhtharcm** [comment]
what did you ended up with?
